instrumentation/otel-webserver-module/src/core/sdkwrapper/SdkHelperFactory.cpp

```diff
@@ -84,9 +88,9 @@ SdkHelperFactory::SdkHelperFactory(
             std::move(sampler)
             ));
 
-    mTracer = mTracerProvider->GetTracer(config->getOtelLibraryName(), libraryVersion);
+    mTracer = mTracerProvider->GetTracer("C++", libraryVersion);
```

Comment:
should be cpp instead? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#telemetry-sdk

libraryVersion should be of opentelemetry-cpp version. Here, it forwards webserver version.